### PR TITLE
task_DEV-2

### PR DESCRIPTION
**Pull request by TatianaGantievskaya**

### CODE REVIEW

File: ModifiedString/EntryPoint.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  class EntryPoint
+  {
+    static void Main(string[] args)
+    {
+      string inputString = null;
+      while (string.IsNullOrEmpty(inputString))
+      {
+        Console.WriteLine("Enter your string: ");
+        inputString = Console.ReadLine();
+        if (inputString.Equals(string.Empty))
```
Comment (IgorKheidorov) [inline ModifiedString/EntryPoint.cs]: inputString.Empty(), more simple

File: ModifiedString/StringProcessing.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  /// <summary>
+  /// This class was created for work with the input string.
+  /// </summary>
+  class StringProcessing
+  {
+    /// <summary>
+    /// This method processes the input string.
+    /// </summary>
+    /// <param name="inputString"> It is a string entered by user. </param>
+    /// <returns> It returns new, modificated, string. </returns>
```
Comment (IgorKheidorov) [inline ModifiedString/StringProcessing.cs]: Not processed, you returns new string

File: ModifiedString/StringProcessing.cs [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    public string ProcessTheString(string inputString)
```
Comment (IgorKheidorov) [inline ModifiedString/StringProcessing.cs]: Processing what? unclear

**Review by IgorKheidorov — CHANGES_REQUESTED** [review]

File: ModifiedString/EntryPoint.cs [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+{
+  /// <summary>
+  /// This is entry point of the program.
+  /// </summary>
+  class EntryPoint
+  {
+    static void Main(string[] args)
+    {
+      string inputString = null;
+      while (string.IsNullOrEmpty(inputString))
```
Comment (AlexanderSoroka) [inline ModifiedString/EntryPoint.cs:13]: Over-complicated solution for such simple task

**Review by AlexanderSoroka — APPROVED** [review]

**Pull request merged**